src/i18n/locales/en/translation.json

```
@@ -570,8 +570,12 @@
       "new_budget": {
         "label": "New budget"
       },
-      "set_budget": {
-        "title": "Set a new budget",
+      "enable_login": {
+        "title": "Enable website login",
+        "subtitle": "Authenticate using <0>lnurlAuth</0>"
```

the text needs some update. 

```suggestion
        "subtitle": "Automatically login without confirmation when the website requests."
```

---

Done!